app/views/carts/show.html.erb

```diff
@@ -21,3 +21,9 @@
 
   Total Before Tax: $<%= @cart.total_price %>
 <% end %></div>
+
+<% if current_user %>
+  <%= button_to "Checkout" %>
+<% else %>
+  <%= link_to "Login or Create Account to Checkout",  login_path %>
```

Comment:
can we do something like:

```
<%= link_to "Login",  login_path %> or <%= link_to "Create Account", new_user_path %> to Checkout"
```
This way, new users only need to click one link instead of 2. Unless the spec specifically wants this?

Reply:
I like this idea (especially in light of the accessibility conversations we had last week), but the spec is specific to the wording of the link!